src/SearchParameters/index.js

```diff
@@ -272,7 +272,7 @@ function SearchParameters(newParameters) {
   /**
    * Enable the ranking informations in the response
    * @see https://www.algolia.com/doc#getRankingInfo
-   * @member {integer}
+   * @member {number}
```

Comment:
The user doesn't need to know about any other values than 1 (wich activates the configuration...). Imho {number} should be used to be consistent in our translation of the rest api into JS and we should add according comments like "set the value to 1 to activate the feature".

Actually jsDoc doesn't care at all about any type. It doesn't understand them, nor make any assumptions. Other tools might but not jsDoc itself.